mne/source_estimate.py

```diff
@@ -529,6 +536,48 @@ def __init__(self, data, vertices=None, tmin=None, tstep=None,
         self._update_times()
         self.subject = _check_subject(None, subject, False)
 
+    def __repr__(self):  # noqa: D105
+        if isinstance(self.vertices, list):
+            nv = sum([len(v) for v in self.vertices])
+        else:
+            nv = self.vertices.size
+        s = "%d vertices" % nv
+        if self.subject is not None:
+            s += ", subject : %s" % self.subject
+        s += ", tmin : %s (ms)" % (1e3 * self.tmin)
+        s += ", tmax : %s (ms)" % (1e3 * self.times[-1])
+        s += ", tstep : %s (ms)" % (1e3 * self.tstep)
+        s += ", data shape : %s" % (self.shape,)
+        return "<%s  |  %s>" % (type(self).__name__, s)
+
+    @property
+    def _vertices_list(self):
+        return self.vertices
+
+    @verbose
+    def save(self, fname, ftype='h5', verbose=None):
```

Comment:
does it mean we write to hdf5 by default also for surface SourceEstimates with this?

Reply:
No, `SourceEstimate` overrides this and only calls `super` if they request `h5`